### PR TITLE
Update installer scripts to also look for "st2-rbac-backend" package and enable enterprise backend

### PR DESCRIPTION
**Pull request by Kami**
This pull request updates installer script to also look for the new ``st2-rbac-backend`` package and enable enterprise RBAC backend by setting ``rbac.backend`` config option to ``enterprise``.

### CODE REVIEW

**Comment by Kami** [comment]
Just a "rcap" from my Slack monologue.

IIRC, at some point we did the ``crudini`` step in ``postinst`` of ``bwc-enterprise`` package (or at least we tried to). I don't recall why we moved it here (probably for consistency with st2-packages?).

I know handling post install data migrations steps and such in ``postinst`` is complex and not something we can easily solve.

Having said that - simply enabling rbac and setting backend config option would be easy since it's an idempotent operation.

This would save us and user some work. User wouldn't need to update st2.conf in case they didn't use the installer script and installed the packages directly and we wouldn't need to update st2docs to let users know they need to update st2.conf to enable RBAC.

**Comment by armab** [comment]
Replied in https://github.com/StackStorm/st2-enterprise-rbac-backend/pull/1#pullrequestreview-220221844

File: scripts/bwc-installer-el6.sh [hunk 1]
```diff
@@ -161,6 +168,7 @@ enable_and_configure_rbac() {
   # Enable RBAC
   sudo yum -y install crudini
   sudo crudini --set /etc/st2/st2.conf rbac enable 'True'
+  sudo crudini --set /etc/st2/st2.conf rbac backend 'enterprise'
```
Comment (Kami) [inline scripts/bwc-installer-el6.sh]: This now also happens inside package post install step, but it also doesn't hurt to keep it here for backward compatibility and consistency (it's an idempotent operation).

**Comment by Kami** [comment]
As mentioned on Slack, since we don't do per-release branching and versioning on this repo - I will update the installer scripts to only install ``st2-rbac-backend`` package when version argument is provided and when version is >= 3.0dev || >= 3.0.0.

**Comment by Kami** [comment]
Confirmed it's working end to end on RHEL and Ubuntu. Will go ahead and merge it.

**Pull request merged**